test/ctia/store_test.clj

```diff
@@ -0,0 +1,74 @@
+(ns ctia.store-test
+  (:require [ctia.store :as sut]
+            [ctia.test-helpers.http :refer [app->APIHandlerServices]]
+            [ctia.test-helpers.fixtures :as fixt]
+            [ctia.test-helpers.core :as helpers :refer [GET POST PATCH]]
+            [ctim.examples.incidents :refer [incident-minimal]]
+            [ctia.store-service :as store-svc]
+            [clojure.test :refer [deftest testing is]]))
+
+(def ident {:login "johndoe"
+            :groups ["Administators"]})
+
+(deftest list-all-pages-test
+  (testing "list all pages shall properly list all results for given entity and parameters. "
+    (helpers/fixture-ctia-with-app
+     (fn [app]
+       (let [{:keys [StoreService] :as raw-services} (app->APIHandlerServices app)
+             lifted (-> StoreService
+                        (select-keys [:read-store
+                                      :write-store])
+                        store-svc/lift-store-service-fns)
+             services (assoc raw-services
+                             :StoreService
+                             lifted)
```

Comment:
This lifting logic can now be removed, `(app->APIHandlerServices app)` should be sufficient.